x-pack/plugins/lists/server/plugin.ts

```diff
@@ -44,6 +51,17 @@ export class ListPlugin {
     core.http.registerRouteHandlerContext('lists', this.createRouteHandlerContext());
     const router = core.http.createRouter();
     initRoutes(router);
+
+    return {
+      getListClient: (apiCaller, spaceId, user): ListClient => {
+        return new ListClient({
+          callCluster: apiCaller,
+          config,
+          spaceId,
+          user,
```

Comment:
not sure I'll have access to the `user` in the executor. I'm assuming I should just pass down `'elastic'` in that case?

Reply:
Correct, if you don't have a user then you should pass down 'elastic' but looking at their code here:
https://github.com/elastic/kibana/blob/master/x-pack/plugins/alerting/server/types.ts#L64

You should have a `updatedBy` string which would be the last user to touch the alerting. You could try `updatedBy` and if null try createdBy and if null default to elastic.

I think we just don't deconstruct those extra arguments yet from the executor but we can and should when looking for the user that "owns" the rule. You won't be doing any "mutation" of a list from an executor so it won't be a big deal really what you push down but being as correct as possible will pay off long term.

Reply:
Oh yeah look at that. Okay I'll pass that down.